### DISCUSSION TITLE
Folder landing page

### DISCUSSION THREAD

**Discussion by biji** [requarks/wiki#4209] [Error / Bug Report]
To create a landing page or home for folder, by naming it like folder name. For example `/Product-Support` , then in navigation tree will be named as it's title

But when clicking a folder in Navigation, the landing page is not opened. And when clicking a landing page in a folder, navigation tree will move to upper level, which it should stay inside folder

**Comment by mko-mx** [comment]
This is still an issue today with v2.5.283, the tree should be expanded to the folder matching the page name.


Here's an example structure, "Engineering" is a virtual folder with multiple pages below:
 - `/engineering` - page to serve as folder home for breadcrumbs
 - `/engineering/best-practices` - page to serve as nested folder home for breadcrumbs
 - `/engineering/best-practices/xyz`

<img width="274" alt="CleanShot 2022-06-13 at 11 28 14@2x" src="https://user-images.githubusercontent.com/8079447/173420772-1ed9d912-d99a-4c0f-bd63-c566d4f46531.png">

1. Opening the folder shows the children as expected.
<img width="269" alt="CleanShot 2022-06-13 at 11 31 20@2x" src="https://user-images.githubusercontent.com/8079447/173420868-191568b5-d10e-49e1-a21d-724921f6f1d1.png">


2. Clicking on the folder home page makes the menu go up one level, so users now have to go into the folder again. In this example, it should display the contents of the "Engineering" folder here.

<img width="432" alt="CleanShot 2022-06-13 at 11 32 14@2x" src="https://user-images.githubusercontent.com/8079447/173421061-b36e8b99-fef2-4cdd-a170-daaa96c98ac2.png">

3. The same happens when going deeper into sub-folders, menu shows one level up. In this example it should be open to show the contents of "Best Practices", not "Engineering".
<img width="567" alt="CleanShot 2022-06-13 at 11 39 31@2x" src="https://user-images.githubusercontent.com/8079447/173422241-65f179b3-eefd-407c-88bf-16851486baae.png">


Hope this helps.

**Reply by johnnybaida** [reply]
Hi, I too have encountered the same problem.
I hope it will be resolved soon.

Thanks for the great work!
